pyrgg/pyrgg.py

```diff
@@ -1254,6 +1255,105 @@ def gml_maker(
     return edge_number
 
 
+def gexf_maker(
+        file_name,
+        min_weight,
+        max_weight,
+        vertices,
+        min_edge,
+        max_edge,
+        sign,
+        direct,
+        self_loop,
+        multigraph):
+    """
+    Create output file in GEXF Format.
+
+    :param file_name: file name
+    :type file_name: str
+    :param min_weight: weight min range
+    :type min_weight: int
+    :param max_weight: weight max range
+    :type max_weight: int
+    :param vertices: number of vertices
+    :type vertices: int
+    :param min_edge : minimum edge number
+    :type min_edge : int
+    :param max_edge : maximum edge number
+    :type max_edge : int
+    :param sign: weight sign flag
+    :type sign: int
+    :param direct: directed and undirected graph flag
+    :type direct: int
+    :param self_loop: self loop flag
+    :type self_loop: int
+    :param multigraph: multigraph flag
+    :type multigraph: int
+    :return: edge_number as int
+    """
+    file = open(file_name + ".gexf", "w")
+    dicts = edge_gen(
+        vertices,
+        min_weight,
+        max_weight,
+        min_edge,
+        max_edge,
+        sign,
+        direct,
+        self_loop,
+        multigraph)
+    edge_dic = dicts[0]
+    weight_dic = dicts[1]
+    edge_number = dicts[2]
+    header = '<?xml version="1.0" encoding="UTF-8"?>\n'
+    header +='<gexf xmlns="http://www.gexf.net/1.2draft" version="1.2">\n'
+    date = datetime.datetime.now().date()
+    meta = '    <meta lastmodifieddate="{0}">\n'.format(date)
+    meta +='        <creator>PyRGG</creator>\n'
+    meta +='        <description>{0}</description>\n'.format(file_name)
+    meta +='    </meta>\n'
+    file.write(header)
+    file.write(meta)
+    directed_flag = str(int(2 - direct))
```

Comment:
Why `directed_flag = str(int(2 - direct))`??!!
I think there is no need to this variable (`directed_flag`)